### PR TITLE
feat: nextjs migration

### PR DESCRIPTION
**Pull request by CristhianF7**

### CODE REVIEW

File: build/Dockerfile [hunk 1]
```diff
@@ -1,43 +1,44 @@
...
+FROM --platform=linux/arm64 node:16-alpine AS runner
+WORKDIR /app
 
-RUN apk add -u bash
+ENV NODE_ENV production
+ENV NEXT_TELEMETRY_DISABLED 1
 
-WORKDIR /usr/share/nginx/html
+RUN addgroup --system --gid 1001 nodejs
+RUN adduser --system --uid 1001 nextjs
```
Comment (converge) [inline build/Dockerfile:29]: that's nice! ⭐

File: components/header/index.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Image from 'next/legacy/image';
+
+const Header = () => {
+  return (
+    <div id="header">
+      <div id="kubefirst-logo">
+        <Image src="/static/kubefirst.png" alt="kubefirst logo" width="288" height="74" />
```
Comment (converge) [inline components/header/index.tsx:8]: is it missing measure type?"288px" ?

File: build/Dockerfile [hunk 3]
```diff
@@ -1,43 +1,44 @@
...
+COPY package.json yarn.lock ./
+RUN \
+  if [ -f yarn.lock ]; then yarn --frozen-lockfile; \
+  else echo "Lockfile not found." && exit 1; \
+  fi
 
-COPY package*.json ./
 
-RUN yarn install
+FROM --platform=linux/arm64 node:16-alpine AS builder
```
Comment (pagottoo) [inline build/Dockerfile:12]: we need to remove the platform and let the docker choose based on the architecture of the node where the runner is running

**Review by pagottoo — CHANGES_REQUESTED** [review]
our Dockerfiles need to be agnostic regarding the architecture

**Review by converge — COMMENTED** [review]
nitpick one: there is a warning on `yarn dev` , maybe it's a good time to change it, since the refactor is touching different points. wdyt?

after `yarn dev`, the page loads ate localhost:3000, but its missing the K-ray logo at the top..

there is another warning in the console on `yarn dev`:
```
/!\ You are using legacy implementaion. Please update your code: use createWrapper() and wrapper.useWrappedStore().
4. withRedux(App) created new store with { initialState: undefined, initialStateFromGSPorGSSR: undefined }
4. withRedux(App) created new store with { initialState: undefined, initialStateFromGSPorGSSR: undefined }
4. withRedux(App) created new store with { initialState: undefined, initialStateFromGSPorGSSR: undefined }
The requested resource isn't a valid image for /static/kubefirst.png received text/html; charset=utf-8
```

**Review by converge — APPROVED** [review]

**Review by pagottoo — APPROVED** [review]
lgtm

**Review by jarededwards — APPROVED** [review]

**Pull request merged**